socrata/sources.py

```diff
@@ -140,14 +140,17 @@ def commit(self, uri, seq_num, byte_offset):
 
 
     def _chunked_bytes(self, file_or_string_or_generator, content_type):
```

Comment:
the name of this is pretty awful already but should we add buffer or something to it?

Reply:
sure why not